examples/js_callback/Cargo.toml

```diff
@@ -7,7 +7,7 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 wasm-bindgen = "0.2"
-yew = { path = "../../packages/yew", features = ["csr"] }
+yew = { path = "../../packages/yew", features = ["csr", "tokio"] }
```

Comment:
Why add tokio here?

Reply:
This is to make `cargo make pr-flow` happy. `use_future` is gated under tokio for non-wasm targets.

Reply:
We should just remove that workflow. See https://github.com/yewstack/yew/issues/2637